File: DTC.Api/Dtos/Board/BoardDto.cs
using DTC.Api.Dtos.Location;

namespace DTC.Api.Dtos.Board
{
    public class BoardDto
    {
        public int Id { get; set; }

        public LocationDto? Location { get; set; }

        public int Number { get; set; }

        public string? Label { get; set; }

        public bool IsActive { get; set; }
    }
}

File: dtc.frontend/src/dtos/board/BoardDto.ts
import z from "zod";
import { locationDtoSchema } from "../location/LocationDto";

export const boardDtoSchema = z.object({
  id: z.number(),
  location: locationDtoSchema.nullable(),
  number: z.number(),
  label: z.string().nullable(),
  isActive: z.boolean(),
});

export type BoardDto = z.infer<typeof boardDtoSchema>;
